packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -832,6 +861,81 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
     });
   }
 
+  /**
+   * Store the keyring encryption key in state, encrypted under the current
+   * encryption key.
+   *
+   * @param keyringEncryptionKey - The keyring encryption key.
+   */
+  async storeKeyringEncryptionKey(keyringEncryptionKey: string) {
```

Comment:
may i know why this function is exposed, any use case for this??

Reply:
the use case is to store the keyring encryption key.
i think it will become clearer when you look at the integration PR: https://github.com/MetaMask/metamask-extension/pull/33735/files